src/domain/safe/entities/__tests__/multisig-transaction.builder.ts

```diff
@@ -43,7 +44,8 @@ export function multisigTransactionBuilder(): IBuilder<MultisigTransaction> {
         appendSlash: false,
       })}", "name": "${faker.word.words()}"}`,
     )
-    .with('proposer', getAddress(faker.finance.ethereumAddress()))
+    .with('proposer', delegate)
+    .with('proposedByDelegate', delegate)
```

Comment:
Nit: I think this wouldn't have any impact on the tests, but to make the instances generated by this builder more accurate: I think the `proposer` and the `proposedByDelegate` can't share the same address, right? Otherwise the transaction wouldn't be proposed by a delegate but by the owner themself and therefore `proposedByDelegate` would be `null`.

Reply:
I adjusted it to have a `proposer` and a `null` `proposedByDelegate` to keep it the same as s in f18756f9d9f79e2f9041fb7434f64b8bcd7cdf6b, as well as adding a specific test in 18d892587ff7e1358e88a955b9655282ef3be41b.